robot_skills/src/robot_skills/arm/grasp_sensor.py

```diff
@@ -0,0 +1,127 @@
+from robot_skills.robot_part import RobotPart
+from tue_manipulation_msgs.msg import GripperCommandGoal, GripperCommandAction
+from tue_msgs.msg import GripperCommand
+
```

Comment:
Insufficient whitespace

Reply:
These commits are inherited from master, but I fixed the white spaces.